packages/search-field/src/components/Menu.js

```diff
@@ -44,21 +44,28 @@ const Menu = ({
   onItemClick,
   highlightedIndex,
   footer
-}) => (
-  <DropdownMenuWrapper onMouseDown={onMenuMouseDown}>
-    <ScrollBox
-      maxHeight={maxMenuHeight}
-      onReachEnd={onScrollReachEnd}
-      ref={innerRef}
-    >
-      {items &&
-        items.map((item, index) => {
-          return renderItem(item, onItemClick, index === highlightedIndex);
-        })}
-      {footer}
-    </ScrollBox>
-  </DropdownMenuWrapper>
-);
+}) => {
+  if (!footer && !items.length) return null;
```

Comment:
I am not super found of bracket-less ifs, but that's a personal preference and prettier is ok with that, so no problem